pkg/orchestrator/evaluation/inmemory_broker.go

```diff
@@ -210,8 +210,10 @@ func (b *InMemoryBroker) EnqueueAll(evals map[*models.Evaluation]string) error {
 func (b *InMemoryBroker) processEnqueue(eval *models.Evaluation, receiptHandle string) error {
 	// If we're not enabled, don't enable more queuing.
 	if !b.enabled {
+		log.Debug().Msgf("broker is not enabled, dropping evaluation %s for job %s", eval.ID, eval.JobID)
 		return nil
 	}
+	log.Debug().Msgf("enqueueing evaluation %s for job %s, triggered by: %s", eval.ID, eval.JobID, eval.TriggeredBy)
```

Comment:
might be worth implementing https://github.com/rs/zerolog/blob/v1.29.1/event.go#L48 for `models.Evaluation`

Reply:
Thanks for the pointer. I'll keep an eye on this if we had to log evaluations in different places, and each with the same log content